### ISSUE TITLE
[SF][A11y][UploadCollection]: Cannot set the group label for the UploadCollection

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents-react#2063]
**Describe the bug**
The uploadCollection is a listbox, but there is no option to set the group aria-label. 

**Isolated Example**
*Please provide an isolated example if possible by forking [this codesandbox](https://codesandbox.io/s/ui5-webcomponents-react-template-29l34).*

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/1614528/131458146-5bd4ed48-3e76-44de-acbc-c9338ada7b3d.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:0.31.15
`@ui5/webcomponents-react` version:0.131.15
Operating System:macOS Big Sur 11.5
Browser:Google Chrome Version 92.0.4515.159 (Official Build) (x86_64)

**Additional context**

**Comment by Lukas742** [comment]
Hi @ericchan1983 

as the `UploadCollection` component is developed by our colleagues in the UI5 Web Components team, I'm going to forward this issue to their repo.
